docs/crocks/Maybe.md

```diff
@@ -0,0 +1,97 @@
+# Maybe
+
+`Maybe c a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Maybe` exposes the following constructor and instances:
+
+| Constructor | Instance |
+|:---|:---|
+| [`Nothing`](#nothing), [`Just`](#just), [`of`](#of) | [`inspect`](#inspect), [`maybe`](#maybe), [`either`](#either), [`option`](#option), [`type`](#type), [`equals`](#equals), [`coalesce`](#coalesce), [`map`](#map), [`ap`](#ap), [`of`](#of), [`chain`](#chain), [`sequence`](#sequence), [`traverse`](#traverse) |
+
+## Constructors
+
+### Nothing
+
+`Maybe m => c -> m c _`
+
+### Just
+
+`Maybe m => a -> m _ a`
+
+### of
+
+`Maybe m => a -> m _ a`
+
+## Instances
+
+### inspect
+
+`() => String`
+
+### maybe
+
+`Maybe m => m c a ~> a -> m a -> a`
+
+### either
+
+`Maybe m => m c a ~> (c -> b) -> (a -> b) -> b`
+
+### option
+
+`Maybe m => m c a ~> a -> m a -> a`
```

Comment:
Ah. So THAT `either` is the either defined in `Maybe` and not the pointfree version. If that makes sense.